src/rpc/blockchain.cpp

```diff
@@ -193,9 +195,11 @@ UniValue getbestblockhash(const UniValue& params, bool fHelp)
     if (fHelp || params.size() != 0)
         throw runtime_error(
             "getbestblockhash\n"
-            "\nReturns the hash of the best (tip) block in the longest block chain.\n"
+            "\nReturns the hash of the best (most recent) block in the longest block chain.\n"
```

Comment:
```suggestion
            "\nReturns the hash of the best (most recent) block in the active block chain.\n"
```
Best to word it like this, as with our penalty system for delayed block submission it's possible that the longest chain is not the active/best chain.